internal/cli/server/config.go

```diff
@@ -67,6 +67,9 @@ type Config struct {
 	// Ancient is the directory to store the state in
 	Ancient string `hcl:"ancient,optional" toml:"ancient,optional"`
 
+	// DBEngine is used to select leveldb or pebble as database
+	DBEngine string `hcl:"dbengine,optional" toml:"dbengine,optional"`
```

Comment:
You'll need to add `DBEngine` in the `DefaultConfig` function below. Use `leveldb` as default for now.